### ISSUE TITLE
Device could not be Register

### ISSUE THREAD

**Issue by amitsinhyudiz** [SayHelloGmbH/progressive-wordpress#54]
I have to allow push notification. but some error throw. "Device could not be Register" what is the actual issue?

**Comment by maxime-killinger** [comment]
Same issue here, I can’t find the problem.

**Comment by Chakmall** [comment]
Hi same here all setting i think is ok but when i want to test it to see if push notification work  the " device could not be registered " and  so no push notif

the remaining part is ok

**Comment by RonAbarbanel** [comment]
Same issue here. Any response/solutions?

**Comment by nico-martin** [comment]
Yes, I'm working on a solution. But it's quite complex: https://github.com/SayHelloGmbH/progressive-wordpress/tree/http-web-push
